Applications/Spire/Source/Charting/ChartView.cpp

```diff
@@ -546,19 +628,9 @@ void ChartView::update_auto_scale() {
     auto_scale_top = std::max(auto_scale_top, candle.GetHigh());
```

Comment:
This is also undefined behavior. Create a task to sanitize all uses of ```std::min``` and ```std::max``` and fix them all in that task instead of this one. There are likely many instances of this.